### PR TITLE
(Sprint-74) Commands to create local user and list roles in an org

### PR DESCRIPTION
**Pull request by anusuyar**
```
Comand to list roles in the existing org
vcd role list
isReadOnly    name                          orgName
------------  ----------------------------  ---------
true          System Administrator              System
true          Multisite System                      System
false         Limited System Administrator  System

Command to create a user TestUser with Role name 'System Administrator'
vcd user create TestUser vmware "System Administrator" -f "Anusuya Rangasmy" -e anusuya.cge@gh.com --enabled

User 'TestUser' is successfully created.

If invalid Role name is provided, Following message will be shown. 
Role 'role_name' does not exist.
```

### CODE REVIEW

File: vcd_cli/user.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+@vcd.group(short_help='work with users in current organization')
+@click.pass_context
+def user(ctx):
+    """Work with users in current organization.
+
+\b
+    Examples
+        vcd user create 'user_name' 'password' 'role_name'
+           create user in the current organization with user_name password and role_name .
+    """
```
Comment (pacogomez) [inline vcd_cli/user.py]: add # NOQA to pass flake8 validation

File: vcd_cli/user.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+
+@vcd.group(short_help='work with users in current organization')
+@click.pass_context
+def user(ctx):
+    """Work with users in current organization.
+
+\b
+    Examples
+        vcd user create 'user_name' 'password' 'role_name'
```
Comment (pacogomez) [inline vcd_cli/user.py]: single quotes can be removed when there is no blank space or special character in the token

File: vcd_cli/user.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+              '--im',
+              required=False,
+              default='',
+              metavar='[im]',
+              help='User\'s im address.')
+@click.option('-E',
+              '--enabled',
+              is_flag=True,
+              required=False,
+              default=False,
```
Comment (pacogomez) [inline vcd_cli/user.py:74]: should we enable the user by default?

File: vcd_cli/user.py [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        client = ctx.obj['client']
+        org_name = ctx.obj['profiles'].get('org_in_use')
+        in_use_org_href = ctx.obj['profiles'].get('org_href')
+        org = Org(client, in_use_org_href, org_name == 'System')
+        role = org.get_role(role_name)
+        if len(role) == 0:
+            raise Exception('Role \'%s\' does not exist.' % role_name)
+        else:
+            role_href = role[0]['href']
+            u = org.create_user(user_name, password, role_href, full_name, description, email, telephone, im,
```
Comment (pacogomez) [inline vcd_cli/user.py]: please make sure this passes flake8 validation

File: vcd_cli/vcd.py [hunk 5]
```diff
@@ -110,4 +110,6 @@ def help(ctx, tree):
     from vcd_cli import vapp  # NOQA
     from vcd_cli import vdc  # NOQA
     from vcd_cli import vm  # NOQA
+    from vcd_cli import role  # NOQA
```
Comment (pacogomez) [inline vcd_cli/vcd.py]: please make sure this passes flake8 validation

File: vcd_cli/role.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
+from __future__ import division
```
Comment (pacogomez) [inline vcd_cli/role.py]: not required

File: vcd_cli/user.py [hunk 7]
```diff
@@ -0,0 +1,140 @@
+from __future__ import division
```
Comment (pacogomez) [inline vcd_cli/user.py]: not required

File: vcd_cli/user.py (hunk 3 above)
Comment (anusuyar) [reply vcd_cli/user.py:74]: In create user api, the default value is false. So I thought it is better to have commands option consistent with the API.

File: vcd_cli/user.py (hunk 7 above)
Comment (anusuyar) [reply vcd_cli/user.py]: Fixed.

File: vcd_cli/user.py (hunk 2 above)
Comment (anusuyar) [reply vcd_cli/user.py]: Fixed.

File: vcd_cli/user.py (hunk 1 above)
Comment (anusuyar) [reply vcd_cli/user.py]: Fixed.

File: vcd_cli/vcd.py (hunk 5 above)
Comment (anusuyar) [reply vcd_cli/vcd.py]: checked. I don't see any failures.

File: vcd_cli/user.py (hunk 4 above)
Comment (anusuyar) [reply vcd_cli/user.py]: checked. I don't see any failures.

File: vcd_cli/role.py (hunk 6 above)
Comment (anusuyar) [reply vcd_cli/role.py]: fixed.

File: vcd_cli/user.py (hunk 3 above)
Comment (pacogomez) [reply vcd_cli/user.py:74]: ok

File: vcd_cli/vcd.py (hunk 5 above)
Comment (pacogomez) [reply vcd_cli/vcd.py]: The validations are disabled here because # NOQA, please make sure the imports are in alphabetical order

Comment (anusuyar) [reply vcd_cli/vcd.py]: Fixed it in https://github.com/anusuyar/vcd-cli/pull/1/commits/cf39eb20a13421e6933f0fa1b586bfa6e8128051

**Review by pacogomez — APPROVED** [review]

**Review by rocknes — APPROVED** [review]

**Pull request merged**